docs/tutorials/fundamentals/viewer.md

```diff
@@ -277,46 +247,30 @@ viewer.add_shapes()
 viewer.add_labels()
 ```
 
-Once added, either in the GUI or via the console, these layers become accessible
-in the layers list section of the GUI and at `viewer.layers`.
+but with empty data. Once added, either in the GUI or via the console, these layers become accessible in the layers list section of the GUI and at `viewer.layers`.
 
-Layers can also be deleted by selecting them and then clicking on the trash
-icon, or by dragging the layers and dropping them into the trash.
+Layers can be deleted by selecting them and clicking on the trash icon, or by dragging the layers and dropping them into the trash.
 
 In the console a layer at index `i` can be removed by:
 
-```python
+```{code-cell} python
```

Comment:
This needs to be changed back - it is not supposed to be executed. And this is what was preventing the other screenshots from being created 🎉 
```suggestion
```python
```